### PR TITLE
Code Refactor

### PR DESCRIPTION
**Pull request by luca-peruzzo**
@kathari00 Hi finally here it is the refactor.
Sorry for wasting your time ( :sweat: ) with the previous one, now it works in the keycloak docker container too (tested v23 and v25).
If you have any concerns about this PR please let me know.
Thanks for your time!

[I will leave this PR in draft untill your approval]

### CODE REVIEW

**Comment by kathari00** [comment]
Hi thanks for checking in. Does the navigation work as well?

What about the locale mode does it start up?

**Comment by luca-peruzzo** [comment]
Yes for both, there is only some warnings on the keycloakify generated assets (some .eot fonts missing) but it compiles and shows the page

File: src/keycloak-theme/keycloak-context.provider.ts
Comment (kathari00) [inline src/keycloak-theme/keycloak-context.provider.ts:1]: I would rather put the providers under common

**Review by kathari00 — CHANGES_REQUESTED** [review]
I want to leave the pipe call wrapped in () as it makes it easier to read for me. 

Thanks for polishing this.

**Comment by luca-peruzzo** [comment]
> I want to leave the pipe call wrapped in () as it makes it easier to read for me.
> 
> Thanks for polishing this.

What do you mean? could you please reference a piece of code where this wrapping is missing?

**Comment by kathari00** [comment]
> > I want to leave the pipe call wrapped in () as it makes it easier to read for me.
> > Thanks for polishing this.
> 
> What do you mean? could you please reference a piece of code where this wrapping is missing?

I want to have it like this
[class]="('sth' | KClass)"

Instead of 
[class]="'sth' | KClass"

**Comment by luca-peruzzo** [comment]
> > > I want to leave the pipe call wrapped in () as it makes it easier to read for me.
> > > Thanks for polishing this.
> > 
> > 
> > What do you mean? could you please reference a piece of code where this wrapping is missing?
> 
> I want to have it like this [class]="('sth' | KClass)"
> 
> Instead of [class]="'sth' | KClass"

Ok, I think it is an automatic correction of prettier cause brackets aren't necessary. I could check of it is possible to disable this rule

**Pull request merged**